### PR TITLE
Soapy bindings for BladeRF

### PR DESCRIPTION
**Pull request by zsoerenm**
This builds the SoapySDR bindings for BladeRF SDR.

Windows is excluded because it depends on `BladeRFHardwareDriver_jll`.

### CODE REVIEW

File: S/SoapyBladeRF/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+cmake -DCMAKE_INSTALL_PREFIX=${prefix} \
+      -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+      -DCMAKE_BUILD_TYPE=Release \
+      ..
+
+make -j${nproc}
+make install
+if [[ "${target}" == *-apple-* ]]; then
+    mv ${libdir}/SoapySDR/modules0.8/libbladeRFSupport.so ${libdir}/SoapySDR/modules0.8/libbladeRFSupport.dylib
```
Comment (giordano) [inline S/SoapyBladeRF/build_tarballs.jl:31]: 1. I recommend using `install -Dvm 0755`
2. Just renaming the file is not enough, the soname is likely wrong

**Review by giordano — COMMENTED** [review]
The licence file is missing

File: S/SoapyBladeRF/build_tarballs.jl (hunk 1 above)
Comment (zsoerenm) [reply S/SoapyBladeRF/build_tarballs.jl:31]: 1. I'm sorry, I'm not a compile export. Does the first mean I shall replace `make install` with `install -Dvm 0755`?
What is the advantage?
2. To be honest I copied this line from the other SoapySDR bindings:
https://github.com/JuliaPackaging/Yggdrasil/blob/master/S/SoapyUHD/build_tarballs.jl#L32 &
https://github.com/JuliaPackaging/Yggdrasil/blob/master/S/SoapyRTLSDR/build_tarballs.jl#L31 etc

**Comment by zsoerenm** [comment]
I added the license file.

File: S/SoapyBladeRF/build_tarballs.jl (hunk 1 above)
Comment (zsoerenm) [reply S/SoapyBladeRF/build_tarballs.jl]: Alright I replaced
```
make install
if [[ "${target}" == *-apple-* ]]; then
    mv ${libdir}/SoapySDR/modules0.8/libbladeRFSupport.so ${libdir}/SoapySDR/modules0.8/libbladeRFSupport.dylib
fi
```
with
```
install -Dvm 755 "libbladeRFSupport.so" "${libdir}/SoapySDR/modules0.8/libbladeRFSupport.${dlext}"
```
Is that correct?

**Comment by zsoerenm** [comment]
I think this ready to be merged.

**Pull request merged**